src/configuration/modules/config/menus/dashboard/index.ts

```diff
@@ -53,11 +54,25 @@ export default {
                 tooltip: opt('Color Picker'),
                 command: opt('sleep 0.5 && hyprpicker -a'),
             },
+            shortcut2: {
+                icon: opt('󰒓'),
+                tooltip: opt('HyprPanel Configuration'),
+                command: opt('hyprpanel toggleWindow settings-dialog'),
+            },
             shortcut3: {
                 icon: opt('󰄀'),
                 tooltip: opt('Screenshot'),
                 command: opt(`bash -c "${SRC_DIR}/scripts/snapshot.sh"`),
             },
+            shortcut4: {
+                icon: opt('󰑊'),
+                tooltip: opt('Record Screen'),
+                command: opt(
+                    isRecording.get() === true
+                        ? `${SRC_DIR}/scripts/screen_record.sh stop "${getRecordingPath().replace(/"/g, '\\"')}"`
+                        : 'INTERNAL_RECORDING_START',
+                ),
```

Comment:
```suggestion
                command: opt('hyprpanel_record'),
```
Let's keep this simple. The component will handle the stopping, starting, etc.